gulp/tasks/pl_scripts.js

```diff
@@ -11,7 +11,9 @@ gulp.task('pl_scripts', ['pl_scripts-lint'], function() {
             './_src/pl/vendor/jquery/jquery-2.1.3.min.js',
             './_src/pl/vendor/jquery/jquery.smooth-scroll.js',
             config.pl_src + '/ui.js',
-            config.pl_src + '/color-contrast.js'
+            config.pl_src + '/size-slider.js',
+            config.pl_src + '/color-contrast.js',
+            './_src/vendor/svg4everybody/svg4everybody.min.js' // polyfill for SVGs in IE9-11
```

Comment:
Cool, please move it, which means:

* creating a new scripts gulp task
* adding a new JS reference in our assets includes
* porting over any vendor files you need for the separate JS file (dependencies on your functions/code)